metaflow/plugins/argo/argo_client.py

```diff
@@ -119,7 +118,7 @@ def trigger_workflow_template(self, name, parameters={}):
     def schedule_workflow_template(self, name, schedule=None, timezone=None):
         # Unfortunately, Kubernetes client does not handle optimistic
         # concurrency control by itself unlike kubectl
-        client = self._kubernetes_client.get()
+        client = self._client.get()
         body = {
             "apiVersion": "argoproj.io/v1alpha1",
             "kind": "CronWorkflow",
```

Comment:
As soon as CronWorkflow is introduced, the need for backfill will be requested.
How do you plan on supporting backfill? With `python flow.py argo backfill --start --end --format` 
- https://argoproj.github.io/argo-workflows/cron-backfill/
- https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/cron-backfill.yaml

Reply:
@talebzeghmi - cron workflows have been supported with Metaflow-on-Argo Workflows for a while. Supporting backfills is on our roadmap too